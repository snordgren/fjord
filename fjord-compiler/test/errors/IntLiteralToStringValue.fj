module IntLiteralToStringValue

stringValue : String
stringValue = 30
