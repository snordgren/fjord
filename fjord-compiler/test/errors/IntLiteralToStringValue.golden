errors/IntLiteralToStringValue.fj:4:15:
  |
4 | stringValue = 30
  |               ^^^
expression has type BuiltIn.Int, expected BuiltIn.String
